src/python/bot/testcase_manager.py

```diff
@@ -670,12 +670,14 @@ def test_for_crash_with_retries(testcase,
                                 testcase_path,
                                 test_timeout,
                                 http_flag=False,
-                                compare_crash=True):
+                                compare_crash=True,
+                                use_gestures=True):
```

Comment:
can we move use_gesture before compare_crash, feels like compare_crash should be last and important for visibility. same in caller.

Reply:
Done!